src/actix/server_request.rs

```diff
@@ -134,6 +135,17 @@ impl HttpRequestExt for HttpRequest {
     }
 }
 
+impl actix_web::FromRequest for Event {
+    type Config = ();
+    type Error = actix_web::Error;
+    type Future = Ready<std::result::Result<Event, actix_web::Error>>;
+
+    fn from_request(req: &HttpRequest, payload: &mut actix_web::dev::Payload) -> Self::Future {
+        // TODO: request_to_event(req, web::Payload(payload.take()))
+        ok(Event::default())
+    }
+}
+
```

Comment:
Can we do this in another PR, after this PR to move the code?

Reply:
Sure thing. I can back it out.